packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -36,7 +36,7 @@
         data-translate="pci_database_add_discovery_project_banner_message"
     ></span>
     <!-- The href link will be added when the discovery project activation page is implemented  -->
-    <a href="#">
+    <a data-ng-click="$ctrl.onActivateProjectClick()" rel="noopener">
```

Comment:
**nitpick**: avoid having link elements without a href attribute (accessibility)
Please replace by a button element with the `oui-button_link` class or a `oui-button` component with a `link` variant property

Reply:
From what I saw in the UI-Kit, the link variant has a hover effect that don't replicate the hover behaviour of an `<a>` element.
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-buttons-webcomponent-ghost--default
Which make it an unsuitable choice to fit the feature request.

I've already suggested to the UI designer to use a button for this component for the same argument that you proposed. But the hyperlink option was the one that was selected.

Reply:
if you prefer to stay with `<a>` you can use ui-sref and the data-track-on / data-track-type / data-track-name

Reply:
Thanks @ghyenne 
I've implemented your suggestion.